expenses/migrations/0001_initial.py

```diff
@@ -1,38 +1,22 @@
+# Generated by Django 4.0.3 on 2022-04-03 19:06
+
 from django.db import migrations, models
 
 
 class Migration(migrations.Migration):
 
     initial = True
 
-    dependencies = []
+    dependencies = [
+    ]
 
     operations = [
         migrations.CreateModel(
-            name="Expenses",
+            name='Expenses',
```

Comment:
Is there a reason behind this change?
Is there a difference between " and ' in python in general? just making sure

Reply:
There isn't a really big difference
you can go and read a bit but it is mostly a code-style
https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python
It was created like it by the Django framework and I didn't touch it